db/migrate/20231005071412_rename_columns_in_like.rb

```diff
@@ -0,0 +1,6 @@
+class RenameColumnsInLike < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :likes, :users_id, :user_id
```

Comment:
- [ ] Please modify this column to be author_id as the foreign key for the users table